reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt

```diff
@@ -107,6 +116,16 @@ class AntennaAttributionDocumentReporter : Reporter {
                         }
                     }
                 }
+
+                if (templatePath.walk().maxDepth(1).filter { it.isFile }.toList().size > 4) {
+                    fontFileNames.associateWithTo(fontFiles) {
+                        templatePath.resolve(it).also { file ->
+                            require(file.isFile && file.extension == "ttf" && file.length() > 0) {
+                                "The file '$file' does not point to a valid font file."
+                            }
+                        }
+                    }
+                }
```

Comment:
I believe we could improve the check here, as just having more than 4 files probably is not a good indicator whether these files might be font files. How about this instead:

                fontFileNames.associateWithTo(fontFiles) { templatePath.resolve(it) }

                val validFontFileCount = fontFiles.values.count { file ->
                    (file.isFile && file.extension == "ttf" && file.length() > 0).also {
                        "The file '$file' does not point to a valid font file."
                    }
                }
                
                require(validFontFileCount == 0 || validFontFileCount == fontFileNames.size) {
                    "Either no or exactly ${fontFileNames.size} valid font files need to be specified."
                }

Reply:
Here's another improved proposal that does not print errors if all font files are absent:

                fontFileNames.associateWithTo(fontFiles) { templatePath.resolve(it) }

                val invalidFontFiles = fontFiles.values.filterNot { file ->
                    file.isFile && file.extension == "ttf" && file.length() > 0
                }

                require(invalidFontFiles.isEmpty() || invalidFontFiles.size == fontFileNames.size) {
                    val fileList = invalidFontFiles.joinToString(", ") { "'$it'" }
                    "The file(s) $fileList do(es) not point to a valid font file."
                }

Reply:
I like the second proposal because it allows to generate the report without passed fonts.

Reply:
Done.